web/index.html

```diff
@@ -1,11 +1,183 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>My CV v2 - starter</title>
+    <title>Antonio Radosav - myCv starter</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport", content="width-device-width, initial scale = 1">
   </head>
   <body>
-    <!-- Change this file -->
-    <h1>Hi! How are you today?</h1>
-    <p>I am the file <code>index.html</code> and you want to change me!</p>
+
+    
+    <grid class="about">
+      <c span="row" height="30"></c>
+      <c span="row">
+        <h1 class="main-title">Antonio Radosav</h1>
+      </c>
+      <c span="row">
+        <h2 class="second-title">Jnior Web Developer</h2>
+      </c>
+      <c span="row" height="30"></c>
+      <c span="row">
+        <h3 class="section-title">About me</h3>
+      </c>
+      <c span="row">
+        <div class="box">
+          <p class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum laudantium molestiae esse, modi aut reiciendis, nobis hic ipsam ut accusantium iure nemo sint sequi mollitia. Dicta dolor quas voluptatem tempora.</p>
+        </div>
+      </c>
+      <c span="row" height="30"></c>
+      <c span="row" class="tCentered">
+        <button class="bAtention">Download my CV</button>
+      </c>
+      <c span="row" height="30"></c>
+      <c span="row" class="tCentered">
+        <button class="circle"><i class="my-cv-arrow-down"></i></button>
+      </c>
+    </grid>
+    
+    
+    <grid class="connect">
+      <c span="row" height="30"></c>
+      <c span="row">
+        <h3 class="section-title">Let's connect</h3>
+      </c>
+      <c span="row"></c>
+      <c span="row">
+        <div class="image-wrapper">
+          <img class="avatar" src="image/antonio.jpeg" alt="">
```

Comment:
you're missing the value of the alt attribute

Reply:
done